packages/app/src/runner/SpecRunnerOpenMode.vue

```diff
@@ -179,21 +172,32 @@ const {
   cleanupRunner,
 } = useEventManager()
 
+const specsListWidthPreferences = computed(() => {
+  return props.gql.localSettings.preferences.specListWidth ?? runnerUiStore.specListWidth
+})
+
+const reporterWidthPreferences = computed(() => {
+  return props.gql.localSettings.preferences.reporterWidth ?? runnerUiStore.reporterWidth
+})
+
+// we must update pr before calling userRunnerStyle, to make sure that values from GQL
```

Comment:
Comment seems incomplete

Reply:
Dang I did not see that test failure, gonna check it out and fix that comment!

Reply:
I was able to see it fail but only once. I think though, we might have run afoul of some gql updates when trying to close the specs list so soon after the doing the last resize earlier in the test - from what I saw the reporter ended up too wide, which could be explained if closing the specs list happened to resolve before updating the width. 

I adjusted the test a bit get better separation between those things. Would be good to know if you can still reproduce? Though I think it represents something no user can every be fast enough to do.